bootstrap/eks/controllers/eksconfig_controller.go

```diff
@@ -294,7 +293,7 @@ func (r *EKSConfigReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Man
 	b := ctrl.NewControllerManagedBy(mgr).
 		For(&eksbootstrapv1.EKSConfig{}).
 		WithOptions(option).
-		WithEventFilter(predicates.ResourceNotPausedAndHasFilterLabel(mgr.GetScheme(), logger.FromContext(ctx).GetLogger(), r.WatchFilterValue)).
+		WithEventFilter(predicates.ResourceHasFilterLabel(mgr.GetScheme(), logger.FromContext(ctx).GetLogger(), r.WatchFilterValue)).
```

Comment:
Subtle change here is that Reconcile will now be called when the EKSConfig is paused, and we'll rely on the "EnsurePaused" condition.

Reply:
No action needed, just stating the change.